document/core/exec/runtime.rst

```diff
@@ -75,14 +75,14 @@ Results
 ~~~~~~~
 
 A *result* is the outcome of a computation.
-It is either a sequence of :ref:`values <syntax-val>`, a :ref:`trap <syntax-trap>`, or an uncaught exception wrapped in its :ref:`throw context <syntax-ctxt-throw>`.
+It is either a sequence of :ref:`values <syntax-val>`, a :ref:`trap <syntax-trap>`, or an :ref:`uncaught exception <exec-throwadm>`.
 
 .. math::
    \begin{array}{llcl}
    \production{result} & \result &::=&
      \val^\ast \\&&|&
      \TRAP  \\&&|&
-     \XT[\val^\ast~(\THROWadm~\tagaddr)]
+     \val^\ast~(\THROWadm~\tagaddr)
```

Comment:
```suggestion
     \XT[(\THROWadm~\tagaddr)]
```
I believe we need to keep the `XT` here, unfortunately, since there is no rule that would reduce it away.